src/core/Jalasoft.TeamUp.Projects.Core/HealthsService.cs

```diff
@@ -9,16 +9,16 @@
 
     public class HealthsService : IHealthsService
```

Comment:
singular for health, please. And if you're gonna rename trough VS, CTRL + R + R it's your friend https://docs.microsoft.com/en-us/visualstudio/ide/reference/rename?view=vs-2019